### DISCUSSION TITLE
Closing browser/Opening new browser problem

### DISCUSSION THREAD

**Discussion by mattlsp** [webdriverio/webdriverio#8206] [Q&A] [answered]
Hi all!
Is there an option to close browser and open new one?
Tried to do it with
await browser.closeWindow();
await browser.newWindow(url);
but all I get is
_ERROR webdriver: Request failed with status 404 due to invalid session id: invalid session id_ 
(**I'm running it on Browserstack**)

I have a testcase to automate which involves clearing cookies, closing browser and opening new one.

**Comment by christian-bromann** [comment, marked as answer]
Please see the docs on [`reloadSession`](https://webdriver.io/docs/api/browser/reloadSession/).

**Reply by mattlsp** [reply]
Tried it already. Browserstack stops test immediately

**Reply by christian-bromann** [reply]
As the docs say, it creates a new session that means in BS a new job. To get a clear browser session this is the only way to do this.

**Reply by mattlsp** [reply]
Thank you
